core/server/data/meta/description.js

```diff
@@ -14,7 +14,7 @@ function getDescription(data, root) {
     } else if (_.contains(context, 'author') && data.author) {
         description = data.author.bio;
```

Comment:
Considering that some people may be using that field to include HTML I think it might be risky using it in a meta tag.